lib/utils/patch.js

```diff
@@ -94,9 +94,15 @@ function createNoUnusedDisableError(ruleId, severity, message, comment) {
             clone.column = loc.start.column + 1
             clone.endLine = loc.end.line
             clone.endColumn = loc.end.column + 1
+            clone.fix = function fix(fixer) {
+                return fixer.removeRange(loc)
```

Comment:
Please remove comma if it exists. This can make invalid syntax. For example:

```
//eslint-disable-line foo, bar, baz
↓↓↓
//eslint-disable-line foo, , baz
```